_posts/2016-06-24-JabRefForum.md

```diff
@@ -0,0 +1,26 @@
+---
+title: New User Forum
+id: JabRefForum
+author: "[JabRef Developers](https://github.com/orgs/JabRef/teams/developers)"
+---
+
+In cooperation with [discourse](http://www.discourse.org) a new forum for JabRef has just been opened!
+
+You can find it by now in main navigation on JabRef.org or directly here: http://discourse.jabref.org
+
+
+
+During the last years we started to move away from sourceforge.net (for various reasons): The code is now managed at GitHub, downloads are handled by FossHub and we are using our own domain JabRef.org for hosting the website and this blog.
+
+However, there we still some things left at sourceforge: 
+Longstanding JabRef user might be aware that there was forum at sourceforge some time ago which has been closed.
+Moreover, there are two mailing lists on for developers, and [one for users](https://lists.sourceforge.net/lists/listinfo/jabref-users).
+
+With the new forum we now again provide an easy to use platform to ask questions (and hopefully to get answers) from the JabRef community.
+Thus, we decided to officially discontinue the support on the mailing list. We'll leave the list open for some time - but the official communcation channels for the future will be the GitHub issue tracker and the discourse forum.
+
+For those of you who like to stick with a mailing-list like behavior we provide you a guide on how to use the discourse forum just like a mailing list.
+
```

Comment:
Yes, of course. This post is not yet completed... and I also have not written this Guide ;)